### PR TITLE
Windows performance:compare

### PR DESCRIPTION
**Pull request by janbiedermann**

### CODE REVIEW

**Comment by janbiedermann** [comment]
Now ready

File: tasks/testing.rake [hunk 1]
```diff
@@ -1,4 +1,8 @@
+require_relative "#{__dir__}/../lib/opal/os"
```
Comment (elia) [inline tasks/testing.rake:1]: nit: also here using __dir__ shouldn't be necessary

**Review by elia — APPROVED** [review]
@janbiedermann thanks for this PR! Looks great, just left a couple of comments for minor changes, can you please try them on windows?

Meanwhile I'm pre-approving 👍

File: tasks/performance.rake [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # A set of tasks to track performance regressions by the CI
 
-require 'opal/util'
+require_relative "#{__dir__}/../lib/opal/util"
+require_relative "#{__dir__}/../lib/opal/os"
```
[the remark this answers is not in the document]
Comment (janbiedermann) [reply tasks/performance.rake:4]: I had to use __dir__ so that it would use the patched version

File: tasks/testing.rake (hunk 1 above)
Comment (janbiedermann) [reply tasks/testing.rake:1]: I had to use __dir__ so that it would use the patched version

**Comment by janbiedermann** [comment]
@hmdne OS abstraction is working great

**Pull request merged**